orc8r/cloud/go/services/certifier/protos/certifier.proto

```diff
@@ -48,6 +48,79 @@ message GetCARequest {
   CertType cert_type = 1;
 }
 
+message TokenList {
+  repeated string tokens = 1;
+}
+
+message User {
+  string username = 1;
+  bytes password = 2;
+  TokenList tokens = 3;
+}
+
+enum Effect {
+  UNKNOWN = 0;
+  DENY = 1;
+  ALLOW = 2;
+}
+
+enum Action {
+  NONE = 0;
+  READ = 1;
+  WRITE = 2;
+}
+
+enum ResourceType {
+  // URI is the relative path to the resource, e.g. "/networks/{network_id}"
+  URI = 0;
+  NETWORK_ID = 1;
+  TENANT_ID = 2;
+}
+
+message Resource {
+  Effect effect = 1;
+  Action action = 2;
+  ResourceType resourceType = 3;
+  string resource = 4;
+}
+
+message ResourceList {
+  repeated Resource resources = 4;
+}
+
+message Policy {
+  string token = 1;
+  ResourceList resources = 4;
+}
+
+message GetPolicyDecisionRequest {
```

Comment:
Nit: separate out into two pseudo-sections: 1 for Req/Res, and 1 for "normal" types